### ISSUE TITLE
Visual Studio Code documentation and the tutorials for Microsoft language extensions with compilers at the step to install a compiler do not show instructions on how to download on package managers other than apt

### ISSUE THREAD

**Issue by The-Gre4test** [microsoft/vscode#237107]
Type: <b>Feature Request</b>

The Visual Studio Code documentation and the tutorials for Microsoft language extensions with compilers at the step to install a compiler do not show instructions on how to download on Linux other than on package managers other than apt.

VS Code version: Code 1.96.2 (fabdb6a30b49f79a7aba0f2ad9df9b399473380f, 2024-12-19T10:22:47.216Z)
OS version: Linux x64 6.12.6-200.fc41.x86_64
Modes:

**Comment by bhavyaus** [comment]
@The-Gre4test did you mean in our documentation?

**Comment by The-Gre4test** [comment]
Yes, and the tutorial that shows up when you download C/C++, and possibly other extensions.

**Comment by bhavyaus** [comment]
@The-Gre4test I'm still unclear. Did you mean docs from our website? Or the walkthroughs which show up inside vscode?

**Comment by bhavyaus** [comment]
@The-Gre4test which step of the walkthrough is this? Could you please include a screenshot or repro steps?
